server/lib/host-metrics.js

```diff
@@ -258,6 +258,29 @@ ORDER BY DATE_TRUNC(:timeUnit, t1."createdAt")`,
   return orderBy(mergedTimeSeries, 'date');
 }
 
+export async function getTotalMoneyManagedTimeSeries(host, { startDate, endDate, timeUnit } = {}) {
+  const hostedCollectives = await host.getHostedCollectives();
+  const ids = hostedCollectives.map(c => c.id);
+
+  const results = await sequelize.query(
+    `SELECT t1."netAmountInCollectiveCurrency" as "_amount", t1."hostCurrency" as "_currency", DATE_TRUNC(:timeUnit, t1."createdAt") as "date"
+FROM "Transactions" as t1
+WHERE t1."CollectiveId" IN (${ids})
+AND t1."kind" IN ('CONTRIBUTION', 'EXPENSE', 'ADDED_FUNDS', 'BALANCE_TRANSFER')
```

Comment:
I don't see the point of filtering the kinds.

Reply:
This one I am not sure. If we take into account platform fee, host fee etc wouldn't the total money managed metric be problematic. Also when I am comparing this numbers with `getTotalMoneyManagedAmount`;

https://github.com/opencollective/opencollective-api/blob/e0a67eb968f94357da3b1257e6d5c4d5879fd1d5/server/lib/budget.js#L192-L198

they don't seem to tally with each other if we include the platform fee and stuff. So I was thinking we should exclude those and only consider these kinds. 🤔

Reply:
> they don't seem to tally

do you have a test or a concrete example to back this?

> if we include the platform fee and stuff

this is an highly imprecise way of formulating it. what is "platform fee and stuff" exactly?

> So I was thinking we should exclude those and only consider these kinds.

how did you pick up this list, what are you filtering exactly, for which reason?

I see no reason to filter any kinds, everything should be counted.

Reply:
> > they don't seem to tally
> 
> do you have a test or a concrete example to back this?

Yes, I am going by the following two tests; 

https://github.com/opencollective/opencollective-api/blob/50e81954a99eb0c7a0180ee876569b1ae3a84293/test/server/graphql/v2/query/HostMetricsQuery.test.js#L119-L145

> > if we include the platform fee and stuff
> 
> this is an highly imprecise way of formulating it. what is "platform fee and stuff" exactly?

Sorry, what I meant was all the transaction kinds excluding, `CONTRIBUTION, EXPENSE, ADDED_FUNDS, BALANCE_TRANSFER`. These are, `HOST_FEE, HOST_FEE_SHARE, HOST_FEE_SHARE_DEBT, PAYMENT_PROCESSOR_COVER, PAYMENT_PROCESSOR_FEE, PLATFORM_FEE, PLATFORM_TIP, PLATFORM_TIP_DEBT` and `PREPAID_PAYMENT_METHOD`.

> > So I was thinking we should exclude those and only consider these kinds.
> 
> how did you pick up this list, what are you filtering exactly, for which reason?

When I test it with these kinds (such as `PLATFORM_TIP` or `HOST_FEE`) the test which I've given above will not pass which is why I thought that these should be excluded.

> I see no reason to filter any kinds, everything should be counted.

Let me think more on this and try to count everything. Maybe I should be missing something here; or maybe my test is somehow wrong. 🤔 🤔

Reply:
Ah, I found the problem here. It seems that I didn't put the host id when considering the collective ID list. I think we need this as we are considering the total money managed by the host in this case. That's why the tests were failing. I've corrected that and removed the kinds as you suggested. Sorry for the confusion. 😢 

https://github.com/opencollective/opencollective-api/blob/7a0082ed943070f417113af2998a44f23e048ee2/server/lib/host-metrics.js#L269